zebra/zebra_l2.c

```diff
@@ -270,7 +279,14 @@ void zebra_l2_bridge_add_update(struct interface *ifp,
 void zebra_l2_bridge_del(struct interface *ifp)
 {
 	/* Unlink all slaves to this bridge */
-	map_slaves_to_bridge(ifp, 0);
+	map_slaves_to_bridge(ifp, 0, false, 0);
```

Comment:
It will be helpful if you define a INVALID/DEFAULT chg_flags as zero and then pass this value to functions.

Reply:
ok, I will use ZEBRA_BRIDGE_NO_ACTION as value per default